### PR TITLE
Actions Amal

### PR DESCRIPTION
**Pull request by averma99**

### CODE REVIEW

**Review by jpperret — APPROVED** [review]

**Review by jpperret — CHANGES_REQUESTED** [review]
The action failed to run. Check the actions tab
"You have an error in your yaml syntax on line 23" - Github

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -15,4 +15,43 @@ jobs:
...
+    container: openjdk
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test PacMan Get Valid Moves
+      run: |
+          cd Projects/P3
+          java -version
+          javac -cp "src/:tests/:tests/junit-4.10.jar" src/*.java tests/*.java
+          java -cp "src/:tests/:tests/junit-4.10.jar" org.junit.runner.JUnitCore TestPacManGetValidMoves
```
Comment (jpperret) [inline .github/workflows/main.yaml:30]: TestPacManValidMoves
(drop get)

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -15,4 +15,43 @@ jobs:
...
+    container: openjdk
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test Ghost Get Valid Moves
+      run: |
+          cd Projects/P3
+          java -version
+          javac -cp "src/:tests/:tests/junit-4.10.jar" src/*.java tests/*.java
+          java -cp "src/:tests/:tests/junit-4.10.jar" org.junit.runner.JUnitCore TestGhostGetValidMoves
```
Comment (jpperret) [inline .github/workflows/main.yaml:43]: TestGhostValidMoves

**Comment by averma99** [comment]
Everything is running properly now!

**Pull request merged**